### PR TITLE
W-14326194 next and prev links

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by dukesphere — DISMISSED** [review]

File: modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc [hunk 1]
```diff
@@ -34,8 +33,7 @@ MuleSoft: Publish API Specification to Exchange
 // Fixed (hn) TODO: Cloud IDE only because it contains hamburger menu. Need to make generic. Plus, take a SS in a lower resolution, or only a part of the screen. Hi-res SS are hard to read in a 13 inch display.
 // Fixed (hn) TODO duke: fix/redo image
 image::publish-to-exchange.png["MuleSoft: Publish API Specification to Exchange highlighted in Command Palette"]
-. Type a project name: +
-* *American Flights API ACB*.
+. Type a project name: *American Flights API ACB*
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc]:
```suggestion
. Type a project name: *American Flights API ACB*
```
I think the project name changed. I believe it was supposed to be **American Flights API** (without ACB), but I wonder if there's a conflict with some screenshots in the file or something.

Comment (hannanelson) [reply modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc]: Yes. The screenshot has the ACB.

Comment (hannanelson) [reply modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc]: We could just take it out. It's superfluous.

File: modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc [hunk 2]
```diff
@@ -34,8 +33,7 @@ MuleSoft: Publish API Specification to Exchange
...
 +
```
Comment (hannanelson) [inline modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc]:
```suggestion
```

File: modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc [hunk 3]
```diff
@@ -34,8 +33,7 @@ MuleSoft: Publish API Specification to Exchange
...
 image::set-asset-name.png["American Flights API option highlighted"]
```
Comment (hannanelson) [inline modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc]:
```suggestion
```

File: modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc (hunk 1 above)
Comment (hannanelson) [inline modules/ROOT/pages/tut-af-design-api-publish-api-spec-to-exchange.adoc]:
```suggestion
. Type a project name: *American Flights API*
```

**Review by dukesphere — APPROVED** [review]
lgtm! Thx for doing this!!

**Pull request merged**